packages/graphql/mixin.core.js

```diff
@@ -14,6 +16,23 @@ function exists(path) {
   }
 }
 
+function contextRelativeExternals({ rootDir }, ...externals) {
+  return function(context, request, callback) {
+    if (externals.includes(request)) {
+      const absolutePath = resolveFrom(context, request);
+      if ((absolutePath.match(/node_modules/g) || []).length > 1) {
```

Comment:
No, we want to count it, because we only want to go into this branch if there are 2 or more occurrences of `node_modules` in the string. Because if there is only one, than it means that the package is directly resolvable from `rootDir`.

Reply:
Another way could be to write `/node_modules.*node_modules/.test(absolutePath)` but not sure if that is any better than the current solution.